MovieMVVM/MovieMVVM/Presentation/DetailsMovie/View/Cells/RelatedMoviesCollectionViewCell.swift

```diff
@@ -35,15 +42,32 @@ final class RelatedMoviesCollectionViewCell: UICollectionViewCell {
         fatalError("init(coder:) has not been implemented")
     }
 
+    // MARK: - Public Methods
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        movieImageView.image = nil
+    }
+
+    // MARK: - Public Methods
+
+    func configure(_ movie: RecommendationMovie, imageService: ImageServiceProtocol) {
+        guard let urlString = movie.posterPath else { return }
+        fetchPhoto(imageService: imageService, urlString: urlString)
+        reloadInputViews()
+    }
+
     // MARK: - Private Methods
 
     private func addSubviews() {
         contentView.addSubview(movieImageView)
     }
 
-    func update(_ movie: RecommendationMovie) {
-        guard let url = movie.posterPath else { return }
-        movieImageView.loadImage(urlImage: url)
+    private func fetchPhoto(imageService: ImageServiceProtocol, urlString: String) {
```

Comment:
```suggestion
    private func fetchPhoto(imageService: ImageServiceProtocol, urlString: String) {
```
в vm вынеси тут должна быть кложура